exercises/hexadecimal/src/lib.rs

```diff
@@ -1 +1,33 @@
+fn parse_hex_digit(c: char) -> Option<i64> {
+    match c {
```

Comment:
although it does not hurt to have it since it's deprecated, this lib.rs doesn't necessarily need to have a complete implementation (and we would want it not to have one if it were non-deprecated)